drivers/watchdog/wdt_cc32xx.c

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2017 Matthias Boesl  <matthias.boesl@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @brief Watchdog (WDT) Driver for CC32xx devices
+ *
+ */
+
+#include <watchdog.h>
+#include <soc.h>
+#include <misc/printk.h>
+
+/* Driverlib includes */
+#include <inc/hw_types.h>
+#include <inc/hw_memmap.h>
+#include <inc/hw_ints.h>
+#include <inc/hw_wdt.h>
+#include <driverlib/rom.h>
+#include <driverlib/pin.h>
+#include <driverlib/rom_map.h>
+
+#define SYS_LOG_DOMAIN "dev/wdt_cc32xx"
+#include <logging/sys_log.h>
+
+/* Note: Zephyr uses exception numbers, vs the IRQ #s used by the CC3200 SDK */
+#define EXCEPTION_WDT0  18  /* (INT_WDT - 16) = (34-16) */
+
+static void (*user_cb)(struct device *dev);
+
+/* Keep reference of the device to pass it to the callback */
+struct device *wdog_r;
+
+static void wdt_cc32xx_enable(struct device *dev)
+{
+	ARG_UNUSED(dev);
+	MAP_WatchdogEnable(WDT_BASE);
+}
+
+static void wdt_cc32xx_disable(struct device *dev)
+{
+
+}
+
+static void wdt_cc32xx_isr(void)
+{
+	MAP_WatchdogIntClear(WDT_BASE);
+
+	if (user_cb != NULL) {
+		user_cb(wdog_r);
+	}
+	while(1);
```

Comment:
in general, is there a clang-format file available? where can I find style rules?